src/proxy.ts

```diff
@@ -49,4 +49,30 @@ function computeApi3ReaderProxyV1Address(
   );
 }
 
-export { computeApi3ReaderProxyV1Address };
+function computeCommunalApi3ReaderProxyV1Address(chainId: ethers.BigNumberish, dapiName: string) {
+  return computeApi3ReaderProxyV1Address(chainId, dapiName, 1, '0x');
+}
+
+function computeDappId(dappAlias: string, chainId: ethers.BigNumberish) {
+  if (!DAPPS.some((dapp) => dapp.alias === dappAlias)) {
+    throw new Error(`dApp with alias ${dappAlias} not registered to the package`);
```

Comment:
So the flow is the dApp applies to us to get a dApp ID assigned. We add the dApp to this package and publish, after which we deploy their proxies and let them verify the addresses. This friction is temporary, `dapps/` will be pre-populated with all known dApps out there once we switch to DEAL.